src/Illuminate/Mail/Transport/ResendTransport.php

```diff
@@ -99,7 +99,7 @@ protected function doSend(SentMessage $message): void
                 'attachments' => $attachments,
             ]);
 
-            throw_if($result['statusCode'] != Response::HTTP_OK, Exception::class, $result['message']);
+            throw_if(isset($result['statusCode']) && $result['statusCode'] != Response::HTTP_OK, Exception::class, $result['message']);
```

Comment:
I guess we should wrap the not equal comparison into parenthesis, as `&&` has a lower precedence than `!=`

```php
isset($result['statusCode']) && ($result['statusCode'] != Response::HTTP_OK)
```

reference: https://www.php.net/manual/en/language.operators.precedence.php